core/overlay_transition.go

```diff
@@ -47,7 +47,7 @@ func OverlayVerkleTransition(statedb *state.StateDB, root common.Hash) error {
 			mpt             = tt.Base()
 			vkt             = tt.Overlay()
 			hasPreimagesBin = false
-			preimageSeek    = migrdb.GetCurrentPreimageOffset(root)
+			preimageSeek    = migrdb.GetCurrentPreimageOffset()
```

Comment:
This PR reverts the actions of the base branch, since it has the concept of "current" set of pointers.